report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
+To ensure the most accurate of the device, it is important to ensure the quality and precision of the chosen hardware to be at its highest.
+Therefore, a test of the motors has been conducted, checking their precision and quality.
+
+\subsection*{Motors}
+To ensure the capabilities of the motors, the tests have to try out combinations of target angles and movement speed, and to what extent precision is reached, considering the braking distance.
+Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in Appendix~\ref{appendix:motortest}.
+
+The terms stable, small correction and huge correction are used to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.
+
+The tests' goal is to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; just in case the motor will go further, it will automatically speed up, in the opposite direction, until reaching its goal, hence the correction.
+
+As the motors are used to adjust the target into the center of the camera-position, a certain precision at a given speed is required.
+This means that the motors used, and the movement speed should be able to stop at a certain angle, from that speed, or have a built-in correction as one of the weaknesses of the tests is, that the movement speed is either 0 or x, and not a varying velocity, which it may or may not be in production.
+
+The expected result and the actual result of the test are not as surprising: The higher the speed, the more correction required.
```

Comment:
ret til: "the more correction *is* required"


"The expected result and the actual result of the test are not as surprising" -> betyder det at relationen mellem de 2 ikke er overraskende? eller at det endelige resultat ikke er?